src/pages/index.js

```diff
@@ -72,13 +144,45 @@ function handleImageClick({ name, link }) {
   popupWithImage.open({ name, link });
 }
 
-function handleAddCardSubmit(formData) {
-  const { title: name, url: link } = formData;
+function handleAddCardSubmit({ title: name, url: link }) {
+  addCardPopup.setSaving(true);
+  api
+    .createCard({ name, link })
+    .then((data) => {
+      const cardElement = createCard(data);
+      cardSection.addItem(cardElement);
+      addCardPopup.close();
+      addModalForm.reset();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      addCardPopup.setSaving(false);
+    });
+}
+
+function handleDeleteButton(card) {
+  deletePopup.card = card;
+  deletePopup.open();
+}
 
-  const cardElement = createCard({ name, link });
-  cardSection.addItem(cardElement);
-  addCardPopup.close();
-  addModalForm.reset();
+function handleAvatarSubmit(evt) {
+  evt.preventDefault();
+  const avatarLink = document.querySelector("#avatar-src").value;
```

Comment:
Only method `_getInputValues` collects all the data of the inputs **inside** `PopupWithForm` and passes it to the call of `submitForm`. So you need to use the collected data rather than the inputs.